articles/most-useful-node.js-packages/index.md

```diff
@@ -0,0 +1,349 @@
+## Most Useful Node.js Packages That you need to Know
+Open-source frameworks that forms the ecosystem of JavaScript back the power of Node.js. As a Node.js developer, it is practically impossible to constantly avoid their advantage. With the help of Node.js packages, as a developer, you will conclusively save a lot of time and develop applications that are light, scalable, high speed with tremendous productivity.
+Let's have a look at the packages that will assist you to extend your inclination to Node.js application development.
```

Comment:
Doesn't flow well. "Let's have a look at packages that will extend your Node.js development skills" perhaps?

Reply:
@kimkimani, I still suggest you look at revising this. The sentence is a bit wordy and could be simplified.